examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,86 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mm = Matrix([[m1,  0],
+             [0,  m2]])
+f = Matrix([-(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+            k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+mm_full = Matrix([[1, 0,  0,  0],
+                  [0, 1,  0,  0],
+                  [0, 0, m1,  0],
+                  [0, 0,  0, m2]])
+f_full = Matrix([u1,
+                 u2,
+                 -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                 k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the rhs of q' = G(q, u, t, r, p). Kinematic equation
+
+G = Matrix([u1, u2])
+
+# Form the rhs of the dynamics equations
+
+RHS = mm_full.inv()*f_full
+
+# Form the remaining input lists for the equations of motion class
+
+coordinates = (x1, x2)
+speeds = (u1, u2)
+
+# Form the kinetic energy to attach to one of the EOM instances as an output
+# equation
+
+KE = 0.5 * (m1*u1**2 + m2*u2**2)
+out_eqns = {"kinetic_energy": KE}
+
+# Initialize the equation of motion class using the three forms accepted by
+# ODEFunctionGenerator
+#    [1] x' = F(x, t, r, p)
+#
+#    [2] M(x, p) x' = F(x, t, r, p)
+#
+#    [3] M(q, p) u' = F(q, u, t, r, p)
+#        q' = G(q, u, t, r, p)
+
+eom1 = eombase.EOM(coordinates, speeds, rhs=RHS, output_eqns=out_eqns)
+eom2 = eombase.EOM(coordinates, speeds, mass_matrix_full=mm_full,
+                   forcing_full=f_full)
```

Comment:
`So I guess we need to decide if System is very general or is System made for multibody systems. `
I would say having a general class that is subclassed would provide a broader base for future functionality.

Having just a single variable `states` would give the class less information overall to work with but would be more compact. If subclasses need to know which states are coordinates and which are speeds they could have a simple input variable for the number of coordinates and assume the first elements that correspond to that number are the coordinates and the rest are speeds. 

`You could even imagine how knowing the units of the states would help improve automatic viz and other things.`

I would think this could be very simply by inputting a list of unit names to either `EOM` or `System` where they positionally correspond to coordinates/speeds in the `states` variable. This solution wouldn't require the coordinates and speeds to be separate variables. Example code
```
eom = EOM(states=[q1, q2, q2, u1, u2, u3], units=['m', 'm', 'mm', 'm/s', 'cm/s', 'mm/s'])
```